scripts/ci/40_e2e.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 set -euxo pipefail
 
-yarn run e2e $1 --headless --production --record --group=$1
+HOST="http://localhost:4200"
```

Comment:
Where/how can we tweak this based on the app if the ports differ?

Reply:
Moved individual project's config to workspace.json.